api/src/main/java/org/asynchttpclient/BoundRequestBuilder.java

```diff
@@ -93,6 +97,16 @@ public BoundRequestBuilder setBody(String data) {
     }
 
     @Override
+    public BoundRequestBuilder setBody(Publisher<ByteBuffer> publisher) {
+        return super.setBody(new ReactiveStreamBodyGenerator(client.getConfig().getExecutorService(), publisher));
+    }
+
+    @Override
+    public BoundRequestBuilder setBody(BodyGenerator bodyGenerator) {
+        return super.setBody(bodyGenerator);
+    }
```

Comment:
I'm going to remove all of those from master:
* [CLJ-126](http://dev.clojure.org/jira/browse/CLJ-126) seems to be related to Java 5, that we don't support.
* [CLJ-259](http://dev.clojure.org/jira/browse/CLJ-259) has been open in 2010. Come on, 5 years to come up with a fix!
* [https://github.com/cch1/http.async.client](http.async.client) has just been upgraded to AHC 1.8. So they won't upgrade to AHC 2 any time soon.
* I just hate it with my guts when people push their own mess to others.